libs/saksbehandling-common/src/main/kotlin/behandling/Revurderingaarsak.kt

```diff
@@ -44,6 +44,7 @@ enum class Revurderingaarsak(
     YRKESSKADE(SAKTYPE_BP_OMS, DevOgProd, skalSendeBrev = true),
     RETT_UTEN_TIDSBEGRENSNING(SAKTYPE_OMS, DevOgProd, skalSendeBrev = true),
     FORELDRELOES(SAKTYPE_BP, DevOgProd, skalSendeBrev = true),
+    FRA_0UTBETALING_TIL_UTBETALING(SAKTYPE_OMS, DevOgProd, skalSendeBrev = true),
```

Comment:
Lurer på om jeg er mer fan av `FRA_INGEN_UTEBETALING_TIL_UTBETALING`. Litt mer lesbart da den 0'en fort er litt vanskelig å lese.

Reply:
Ja, ser den. Gjekk for dette sia vi jo snakkar om `nullutbetaling`, ikkje _ingen utbetaling_, så lurer på om det er mindre forvirrande lell sånn. Kanskje `FRA_0-UTBETALING...`?